### PR TITLE
PHPStan update

### PR DESCRIPTION
**Pull request by tomasfejfar**
Jira: N/A
Connection PR: N/A

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated
- [x] You declared if there is a BC break or not (will affect next release of a client)

## Release

- [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

---

### CODE REVIEW

File: tests/FileStorageGcsTest.php [hunk 1]
```diff
@@ -21,8 +22,16 @@ class FileStorageGcsTest extends ClientTestCase
 
     public function setUp(): void
     {
-        $this->credentials = json_decode((string) TEST_GCS_KEYFILE_JSON, true, 512, JSON_THROW_ON_ERROR);
-        $this->rotateCredentials = json_decode((string) TEST_GCS_KEYFILE_ROTATE_JSON, true, 512, JSON_THROW_ON_ERROR);
+        $credentials = json_decode((string) TEST_GCS_KEYFILE_JSON, true, 512, JSON_THROW_ON_ERROR);
```
[the remark this answers is not in the document]
Comment (jirkasemmler) [reply tests/FileStorageGcsTest.php:25]: kdyz tam nechame ten JSON_THROW_ON_ERROR , tak pro spatne nastavene credentials nam to nijak nepomuze (bude to stale strilet Syntax error). A kdyz tam dale delame check na null, tak pro treba ciselnou hodnotu to pro zmenu nespadne

**Review by jirkasemmler — CHANGES_REQUESTED** [review]

File: tests/FileStorageGcsTest.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (tomasfejfar) [reply tests/FileStorageGcsTest.php]: Dal jsem tam jen docblock, nedává smysl se s tím nějak frcat

File: tests/FileStorageGcsTest.php [hunk 2]
```diff
@@ -15,14 +16,19 @@ class FileStorageGcsTest extends ClientTestCase
...
     private array $credentials;
 
     private array $rotateCredentials;
 
     public function setUp(): void
     {
-        $this->credentials = json_decode((string) TEST_GCS_KEYFILE_JSON, true, 512, JSON_THROW_ON_ERROR);
-        $this->rotateCredentials = json_decode((string) TEST_GCS_KEYFILE_ROTATE_JSON, true, 512, JSON_THROW_ON_ERROR);
+        /** @var mixed[] $credentials */
+        $credentials = json_decode((string) 'Xxx', true, 512, JSON_THROW_ON_ERROR);
```
Comment (jirkasemmler) [inline tests/FileStorageGcsTest.php:27]: xxx ? tady by mely zustat puvodni konstanty

Comment (tomasfejfar) [reply tests/FileStorageGcsTest.php:27]: Ježišmarjá, jsem dement, děkuju. Omylem jsem commitnul test (jestli ten phpstan fail je kvuli tomu, že je to ENV).

**Pull request merged**